pegasus/migrations/106_add_popularityrank_to_tutorials.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :popularityrank, Integer, null: false
```

Comment:
The tutorials table has quite a few of the original fields without _, and since this is closest to displayweight, it seemed more consistent to not have one.